packages/components/cypress/e2e/logo.cy.ts

```diff
@@ -49,14 +49,14 @@ describe('logo', () => {
       cy.getComponent('logo', LOGO_ID, 'height');
     });
 
-    it('should not fit the parent height', () => {
-      cy.get('@logo')
-        .parent()
-        .invoke('innerHeight')
-        .then(parentHeight => {
-          cy.get('@logo').invoke('outerHeight').should('not.eq', parentHeight);
-        });
-    });
+    // it('should not fit the parent height', () => {
+    //   cy.get('@logo')
+    //     .parent()
+    //     .invoke('innerHeight')
+    //     .then(parentHeight => {
+    //       cy.get('@logo').invoke('outerHeight').should('not.eq', parentHeight);
+    //     });
+    // });
```

Comment:
Why is this commented out?
If we don't need this this anymore, please delete it. We can always restore it from the history if necessary.

Reply:
yes, this was by mistake. I ll revert the file.